Production Release as on 02-24-2021

rabbi08 opened the pull request:
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):_

<!--### 🚨Before submitting this pull request🚨:-->

<!--_Please do **NOT** submit this PR if you have not done the following:_-->

- [ ] I have checked that no one else is working on similar changes.
- [ ] I have read the latest [**README**](README.md) and understand the development workflow.
- [ ] The name of this branch is: **`feature/<branch_name>`** for a regular feature, **`hotfix/<branch_name>`** for a hotfix, and **`service/<branch_name>`** for a microservice, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [ ] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [ ] There are no merge conflicts.
- [ ] There are no console warnings and errors.
- [ ] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [ ] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

adorapham committed:
Adding the validaiton for Croatian, Greek,Hindi,Hungarian, and Marathi language (#2206)

sujatha-m committed:
About Us is getting overlapped behind the search box while translatin… (#2207)

* About Us is getting overlapped behind the search box while translating for few languages fixed

* moved the searchbar to the right

sujatha-m committed:
Policy pages lack visible left and right margins fixed on small devices (#2211)

hanzala-sohrab committed:
Modified wordings of some sentences in Hindi & English (#2190)

emanuelquerty committed:
#2181- Fix inconsistencies in font in Greek Translation (#2182)

* Adds sans-serif font-family (fallback), which was missing in CSS rules

* Adds sans-serif fallback font to fix font inconsistencies

dependabot[bot] committed:
Bump socket.io from 2.3.0 to 2.4.0 in /backend (#2163)

Bumps [socket.io](https://github.com/socketio/socket.io) from 2.3.0 to 2.4.0.
- [Release notes](https://github.com/socketio/socket.io/releases)
- [Changelog](https://github.com/socketio/socket.io/blob/2.4.0/CHANGELOG.md)
- [Commits](https://github.com/socketio/socket.io/compare/2.3.0...2.4.0)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

linadinh2602 committed:
fix error not show up (#2199)

Jameel139 reviewed (approved):
Looks good.

jessica-lau reviewed (approved):
LGTM. Retested #2206.

![image](https://user-images.githubusercontent.com/56090896/108920754-3c163580-7603-11eb-859d-1ecf466900e2.png)

aditilamkhede committed:
Add work remotely option in Create Post and Filter Page (#2165)

* Add work reotely option in Create Post and Filter Page

Add work reotely option in Create Post and in Filter on feed Page.

* added validation for remote selection

Added validation for remote dropdown value selection(i.e. making it required field) in CreatePost Form.
Added remote field in getPostsSchema.

* Change for older posts without remote

Change for older posts without remote option, i.e. database records/post which do not have remote option as it was added later. Make default option for those posts as 'No'.

* Changed removed hard code for default option

Changed removed hard code for default option from 'No' to options from createPostSettings.

* Changed Hard coded value

* Add and Hide Remote Work Label

hide the 'Remote Work' label from 'Create Post' modal.
For new post-> if user selects 'Yes' to qs 'can work remotely' then the post should have 'Remote Work' label

* Apply/ClearFilter and add RemoteWork label to FilterList

Apply/ClearFilter functionality and added RemoteWork label to FilterList

* Removed Hardcoded text

* Checkbox style

* Update Remote feature with new Changes

Update Remote feature with new Changes. 3 Options on Create Person In-person, Remote, Both. Changes in filter on Helpborad page.

* Changes for old post and label changes

Changes for old post - post with 'Remote Work' tag should be displayed as 'Remote' if fullfillment is null else use fullfillment value.
post with no 'Remote Work' tag and fullfillment is null then display as "Both"
Label changes - Changed label text to Post Fullfillment on filter sidebar, added label on Create Post under Post fullfillment.

* Asiigned ids for gtm tag

Assigned ids to postfillment in filter section on help board and to fullfillment dropdown on create post page.

* Changes for mobile view and refactor code.

* moved context menu (elipsis) to next line for mobile view

* Update Post.js

kamarenmccord committed:
Feature/2192 organistation preformance issue (#2196)

* changed styles for support button

* changed default endpoint

* changed default endpoint to enhance preformance

* removed a bitwise test

* rolled back global styles as this is for a different issue

* changed organization search on frontend to match new backend

TimeaKovacs86 reviewed (approved):
Looks good to me!
